File: C/Assignment24/Program1.c
/////////////////////////////////////////////////////////////////////////////////////////////////////////
//  
// File name :   Program1.c
// Description : Write  program which accept string from user & count number of capital characters.
// Author :      Pallavi Omprakash Malewar
// Date :        21/07/2026
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////

/*

Input :     "Marvellous Multi OS"

Output :    4

*/


#include<stdio.h>
#include<stdlib.h>

int CountCapital(char *str)
{
    int iCnt = 0;

    if(str == NULL) return -1;

    while(*str != '\0')
    {
        if(*str <= 'Z' && *str >= 'A')
        {
            iCnt++;
        }
        str++;
    }
    return iCnt;
}

int main()
{
    char arr[20];
    int iRet = 0;

    printf("Enter the String : ");
    scanf("%[^'\n']s", arr);

    iRet = CountCapital(arr);
    printf("%d\n", iRet);

    return 0;
}

File: JAVA/Assignment 9/Program1.java
/////////////////////////////////////////////////////////////////////////////////////////
//                                                                                     //
// Problem Statement : Write Java program which accept array of characters from user & //
//                     replace each capital character with its corresponing small char.//
// Author : Pallavi Omprakash Malewar                                                  //
// Date : 3 Aug 2026                                                                   //
//                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////


/*  

    Input : b   N   j   B   R   b   A   d   G   G

    Output: b   n   j   b   r   b   a   d   g   g

*/


import java.util.Scanner;

class MyArray
{
    public void ArrayReplace(char Arr[])
    {
        int n = Arr.length;
        int i = 0;

        for(i = 0 ; i < n ; i++)
        {
            if(Arr[i] <= 'Z' && Arr[i] >= 'A')
            {
                Arr[i] = (char)(Arr[i] + ('a' - 'A'));
            }
        }
    }
}

class Program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the Size of Array : ");
        int No = sobj.nextInt();

        char Arr[] = new char[No];

        for(int i = 0 ; i < No ; i++)
        {
            Arr[i] = sobj.next().charAt(0);
        }

        MyArray mobj = new MyArray();

        mobj.ArrayReplace(Arr);

        for(int i = 0 ; i < No ; i++)
        {
            System.out.println(Arr[i]);
        }
    }
}
